packages/robo/src/cli/utils/cli-handler.ts

```diff
@@ -179,6 +236,15 @@ export class Command {
 
 			// If arg is not an option or a subcommand, treat as a positional argument
 			positionalArgs.push(arg)
+
+			// if subcommand is invalid show a message to inform the user.
+			if (!subCommand) {
```

Comment:
This conditional is flawed. The reverse is already executed a few lines above, so I'm concerned this may lead to incorrect short circuits. Specifically with positional arguments.

Please test the `robo build` command's positional arguments to verify this:
`robo build file1.ts file2.ts`

This is also used in the new `add` and `remove` commands to specify package names. I fear that this check will incorrectly exit early for those.

It may be worth applying this logic only to the root `robo` command, or adding a new convention allowing commands to specify whether they use positional arguments or not, similar to Commander.js.